content/product-engineering/engineering/cloud/growth-and-integrations/index.md

```diff
@@ -0,0 +1,199 @@
+            <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/erzhan.png" alt="Picture of Erzhan Torokulov" style="background: transparent; width:128px;"/>
+          </a>
+        </div>
+        <div style="text-align: center;">Erzhan Torokulov</div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <a href="../../../../team/index.md#tharuntej-kandala" target="_blank" rel="noopener">
+            <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/tj.png" alt="Picture of TJ Kandala" style="background: transparent; width:128px;"/>
+          </a>
+        </div>
+        <div style="text-align: center;">TJ Kandala</div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/join-us-vermillion.png" alt="Picture with CTA: Join Us" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">P.S.</div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer</div>
+        <div style="text-align: center; font-size: 12px;">(starting 01/10)</div>
+      </div>
+    </div>
+
+<div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/rob-rhyne.png" alt="Picture of Rob Rhyne" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">Rob Rhyne</div>
+        <div style="text-align: center; font-size: 12px;">Temporary product manager and product designer, growth</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/join-us-vermillion.png" alt="Picture with CTA: Join Us" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">Stephen Gutekanst </div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer (growth)</div>
+      </div>
+    </div>
+  </div>
+
+</section>
+
+## Growth vision
+
+Sourcegraph is optimized to help users discover how powerful the tool is, as quickly as possible. This optimization leads to introductions to new users and customers. It also increases usage at customer accounts and with individual developers. By addressing these issues we will:
+
+- Generate product qualified leads for the business
+- Expand usage in existing accounts
+- Increase the frequency of all users from MAU to DAU
+- Imrprove retention of all users
```

Comment:
```suggestion
_Sourcegraph is optimized to help users discover how powerful the tool is, as quickly as possible._

This vision is necessarily broad because the growth-oriented goal for the team is the generation of Product Qualified Leads. These PQLs will generally be generated from users who find value in the product and integrate Sourcegraph into their daily routines. As such, this team will also impact the following objectives:

- Improve the retention of all users
- Increase the frequency of users from monthly active to daily active
- Expand usage in existing accounts
```